src/data/skill-name.csv

```diff
@@ -33,15 +33,17 @@
 "Shadow Break","Shadow Break"
 "汝、皇帝の神威を見よ","Thou, witness the heavenly might of the Emperor"
 "ブレイズ・オブ・プライド","Blaze of Pride"
+"尊み☆ﾗｽﾄｽﾊﾟ—(ﾟ∀ﾟ)—ﾄ!","Kyuute☆Last SpuU(ﾟ∀ﾟ)Uurt!"
 "アングリング×スキーミング","Angling×Scheming"
 "∴win Q.E.D.","∴win Q.E.D."
 "ひらめき☆ランディング","Inspiration ☆ Landing"
 "G00 1st.F∞;","G00 1st.F∞;"
 "レッツ・アナボリック！","Let's anabolic!"
+"I'M☆FULL☆SPEED!!",""
 "ブルーローズチェイサー","Blue Rose Chaser"
 "U=ma2","U=ma2"
 "勝利のチケットを、君にッ！","The ticket to victory, for you!"
-"Schwarze Schwert","Schwarze Schwert"
+"Schwarzes Schwert",""
```

Comment:
Needs fixing

Reply:
How did this error occur? Is this not pulled with the import script directly from master.mdb?

Reply:
Probably because Cygames fixed the original string

Reply:
It was pulled using the import script, yes. I don't think this is an error.
I've cleared my whole master folder and let it download again just now to make sure. It is like this in the original.
I believe they changed it. There were a bunch of other changes/fixes they seem to have made in a recent update, judging by other files I've worked on.

As for the lack of `translation` column, the db update script ignores entries with empty tl so I am just leaving those alone currently when no tl needs to be made. Don't know the policy here as I've seen instances of both (empty/copy).

Reply:
They did indeed fix this a while ago, yea (woo, it's now grammatically correct german!).